packages/config/esbuild/static/cli.js

```diff
@@ -0,0 +1,14 @@
+const cli = {
+  bundle: true,
+  entryPoints: [
+    './src/main.ts',
+  ],
+  format: 'cjs',
+  minify: true,
+  outfile: './dist/cli/index.js',
+  platform: 'node',
```

Comment:
is node here because  of esm version

Reply:
Since this is for building a CLI execution app, which can only occur in node, node is used.